docker/data-generator/data_generator.py

```diff
@@ -0,0 +1,32 @@
+import boto3
+import os
+
+MINIO_BUCKET = "data"
+MINIO_ENDPOINT = "http://minio:9000"
+MINIO_ACCESS_KEY_ID = "minio"
+MINIO_SECRET_ACCESS_KEY = "minio123"
+
+BOTO3_CLIENT = boto3.client(
+    "s3",
+    endpoint_url=MINIO_ENDPOINT,
+    aws_access_key_id=MINIO_ACCESS_KEY_ID,
+    aws_secret_access_key=MINIO_SECRET_ACCESS_KEY,
+)
+
+
+def main():
+    root_file_path = "./data"
+    file_name_list = ["2019-Oct.csv", "2019-Dec.csv"]
+
+    for file_name in file_name_list:
+        local_csv_file_path = os.path.join(root_file_path, file_name)
```

Comment:
```suggestion
        local_csv_file_path = f"./data/{file_name}"
```
이것도 이렇게 변경할 수 있을 것 같아요~

Reply:
이렇게 되면 root_file_path 변수는 삭제해도 되겠네요!

Reply:
음 저는 하은님이 작성 해주신 것 처럼 명시적으로 남겨 두는 것이 좋아 보입니다. 누군가 코드를 사용하려고 봤을 때 더 직관적인 것 같아서 그렇습니다!

Reply:
f-string으로 쓰는게 더 명시적이고 직관적이지 않나요?? `os.path.join(root_file_path, file_name)` 으로 쓰게 되면 `root_file_path` 같은 경우, 그 변수를 또 찾아서 시선을 옮겨야하는데, f-string을 쓰면 바로 그 string이 직관적으로 보여서요! 두 분 의견은 어떠신가요~?

Reply:
개인적인 선호를 따지자면 전 f-string을 사용하지 않고 기존의 방법대로 한 뒤, `root_file_path`를 `path.py` 같이 한 파일에서 고정 경로를 모아두고 관리하는걸 선호하기는 합니다!
하지만 현재 상태에서 path가 여러개인 상황이 아니다보니, 이 코드에서는 f-string을 사용하는게 더 직관적으로 보이기는 합니다.

Reply:
오 좋습니다! 그러면 이 부분은 구현하신 하은님 취향대로 하시죠!

Reply:
resolve할게요~